### PR TITLE
OCPBUGS-23406: Enable AllNamespaces install mode

### PR DESCRIPTION
**Pull request by anirudhAgniRedhat**
Adding support for 'All Namespace' mode on install time.

### CODE REVIEW

**Review by lunarwhite — COMMENTED** [review]
/lable qe-approved

File: bundle/manifests/cert-manager-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -735,7 +735,7 @@ spec:
     type: SingleNamespace
```
[the remark this answers is not in the document]
Comment (arkadeepsen) [reply bundle/manifests/cert-manager-operator.clusterserviceversion.yaml:735]: I think if we currently support this option then there is a chance that some customer may use this option (installed using CLI method).

We need to give some notice to the customers before deprecating any options currently supported.

**Comment by swghosh** [comment]
@lunarwhite Not sure why but the qe-approved label seems missing despite yours adding it in the past.

**Pull request merged**